wiki/BBCode/en.md

```diff
@@ -1,581 +1,417 @@
+
+**BBCode** is a markup syntax that is used in the osu!forums and, to a larger extent, the vast majority of forums on the internet to enable rich text formatting.
+It is made up of tags that surround the text to enrich and, sometimes, attributes.
+
+In the osu!website, BBCode is used in forum posts, signatures and userpages.
+This article is a listing of the available tags.
+
+## Notes
+
+Know that the osu!fourms gives you a few format buttons that you could use; however, you could just type them in yourself.
+
+BBCode tags are case insensitive.
+
+### BBCode Button Behavior
+
+If you highlight the text that you want to format then click on one of them, e.g. bold, italics, strikethrough, or underline, it will add said format code to the highlighted text automatically.
+
+If you click the button without highlighting text, it will surround the tags around your text cursor.
+
+### Combining tags
+
+You can combine tags to create an even richer formatting.
+
+When combining tags, you **must** respect the order/nesting of the tags.
+Failing to follow this may break your entire forum post, signature, or userpage.
+
+For example (pay attention to the order of the code):
+
+- `[centre]` `[b]` _text_ `[/b]` `[/centre]` is correct, but
+- `[b]` `[centre]` _text_ `[/b]` `[/centre]` is wrong.
+
+## List of BBCode Tags
+
+### Bold
+
+This will help emphasize words or an entire paragraph in a heavier way than [italics](#italic).
+
+- Dedicated button: ![Bold button](img/bold.jpg "Bold button")
+- Notes:
+  - Use this tag sparingly.
+  - Overuse of this will create discomfort for the reader's eyes.
+- Syntax:
+```prolog
+[b] ... [/b]
+```
+
+### Italic
+
+This will help emphasize words or an entire paragraph in a lighter way than [bold](#bold).
+
+- Dedicated button: ![Italic button](img/italic.jpg "Italic button")
+- Notes:
+  - Use this tag sparingly.
+  - Overuse of this will weaken the emphasize overall.
+- Syntax:
+```prolog
+[i] ... [/i]
+```
+
+### Underline
+
+This will help emphasize the importance of a word(s) or an entire paragraph.
+
+- Dedicated button: ![Underline button](img/underline.jpg "Underline button")
+- Notes:
+  - Use this tag sparingly.
+  - Overuse of this will create discomfort for the reader's eyes.
+  - Overuse of this will weaken the emphasize overall.
+- Syntax:
+```prolog
+[u] ... [/u]
+```
+
+### Strike
+
+_Also known as **strikethrough**._
+
+This will cross out an incorrect word(s) or an entire paragraph.
+
+- Dedicated button: ![Strike button](img/strike.jpg "Strike button")
+- Notes:
+  - Use this tag sparingly.
+  - Overuse of this will create discomfort for the reader's eyes.
+- Syntax:
+```prolog
+[strike] ... [/strike]
+```
+
+### Colour
+
+_BBCode calls this "Color"._
+
+This will give colour words or an entire paragraph.
+
+- Dedicated buttons:
+![The colour box](img/colour.jpg "The colour box")
+- Notes:
+  - This is located on the right side of the text box.
```

Comment:
will replace with `The colour box is`... instead (starting with `located on the `... sounds kinda weird)